packages/svelte-ux/src/lib/utils/dateInternal.ts

```diff
@@ -0,0 +1,11 @@
+import { DayOfWeek } from './date_types';
+
+export function getWeekStartsOnFromIntl(locales?: string): DayOfWeek {
+  if (!locales) {
+    return DayOfWeek.Sunday;
+  }
+
+  const info = new Intl.Locale(locales);
+  // @ts-ignore
+  return (info.weekInfo.firstDay ?? 0) % 7; // (in Intl, sunday is 7 not 0, so we need to mod 7)
+}
```

Comment:
I don't know why, but if I put this function in `date.ts`, I can't call it from `locale.ts`!

I have a message like: `getWeekStartsOnFromIntl is not a function.`

Reply:
hmm, that is odd.  I'll try to look at it locally, but we can merge as is for now.